pkg/webhook/policy.go

```diff
@@ -103,6 +103,8 @@ const (
 	denyResponse    admissionReqRes = "deny"
 	allowResponse   admissionReqRes = "allow"
 	unknownResponse admissionReqRes = "unknown"
+
+	namespaceStr string = "Namespace"
```

Comment:
is the `string` type necessary?

Reply:
this was due to linter complaining, reverted since we don't need this anymore